sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -299,4 +310,19 @@ public boolean isAllowedGroup(String group) {
                        .orElseGet(Collections::emptyList)
                        .contains(group);
     }
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
+        validateDomain();
+    }
+
+    private String addSlash(String uri) {
+        return uri.endsWith("/") ? uri : uri + "/";
+    }
+
+    private void validateDomain() {
```

Comment:
`validateGraphUri` or `validate` is more reasonable.
Furthermore, can we just delete this function, and write validate related code in `afterPropertiesSet` directly?